monitoring/README.md

```diff
@@ -6,112 +6,135 @@
+```bash
+$ kubectl -n ks-monitoring get pods
+NAME                                                  READY   STATUS    RESTARTS   AGE
+grafana-7969f6cf5c-lp4g6                              1/1     Running   0          66s
+prometheus-kube-prometheus-operator-b56d747c9-wfs2w   1/1     Running   0          42s
+prometheus-prometheus-kube-prometheus-prometheus-0    2/2     Running   0          39s
+pyroscope-0                                           1/1     Running   0          90s
+pyroscope-alloy-0                                     2/2     Running   0          90s
+```
 
-2. Configure Prometheus and Pyroscope to scrape your KubeStellar spaces:
+#### 2. Configure Prometheus and Pyroscope to scrape your KubeStellar spaces:
 
-    a) WDS space:
+a) WDS space:
 
-    ```bash
-    $ ./configure-metrics-wds.sh
-    ```
+```bash
+$ ./configure-metrics-wds.sh
+```
 
-    If you have more than one WDS, you can run the above script with the `--space-name` flag. For example: `./configure-metrics-wds.sh --space-name wds2`
-    
-    Optionally, check the prometheus service monitor objects for the WDS space:
+If you have more than one WDS, you can run the above script with the `--space-name` flag. For example: `./configure-metrics-wds.sh --space-name wds2`
 
-    ```bash
-    $ kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack"
-    NAME                           AGE
-    wds1-apiserver                 64m
-    wds1-ks-controller-manager     64m
-    wds1-ks-transport-controller   64m
-    ```
+Optionally, check the prometheus service monitor objects for the WDS space:
 
-    b) ITS space:
+```bash
+$ kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack"
+NAME                           AGE
+wds1-apiserver                 64m
+wds1-ks-controller-manager     64m
+wds1-ks-transport-controller   64m
+```
 
-    ```bash
-    $ ./configure-metrics-its.sh
-    ```
+b) ITS space:
 
-    If you have more than one ITS, you can run the above script with the `--space-name` flag. For example: `./configure-metrics-its.sh --space-name its2`
-    
-    Optionally, check the prometheus service monitor objects for the ITS space:
+```bash
+$ ./configure-metrics-its.sh
+```
 
-    ```bash
-    $ kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack"
-    NAME                           AGE
-    its1-apiserver                 64m
-    its1-status-addon-controller   64m
-    ```
+If you have more than one ITS, you can run the above script with the `--space-name` flag. For example: `./configure-metrics-its.sh --space-name its2`
 
+Optionally, check the prometheus service monitor objects for the ITS space:
 
-3. Connect to Grafana UI: 
+```bash
+$ kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack"
+NAME                           AGE
+its1-apiserver                 64m
+its1-status-addon-controller   64m
```

Comment:
This is still smaller than the listing I get:

```shell
mspreitz@mjs13 monitoring % kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack"  
NAME                           AGE
its1-apiserver                 10s
its1-status-addon-controller   10s
wds1-apiserver                 27s
wds1-ks-controller-manager     29s
wds1-ks-transport-controller   28s
```

Reply:
Fixed!